doc/api_methods.md

```diff
@@ -233,122 +244,121 @@ Used for [Torque maps](https://github.com/CartoDB/torque). Note that it does not
 Used to create an animated torque layer with customized settings.
 
 ```javascript
-  // initialize a torque layer that uses the CartoDB account details and SQL API to pull in data
-  var torqueLayer = new L.TorqueLayer({
-    user       : 'viz2',
-    table      : 'ow',
-    cartocss:  CARTOCSS
-  });
+// initialize a torque layer that uses the CartoDB account details and SQL API to pull in data
+var torqueLayer = new L.TorqueLayer({
+  user : 'viz2',
+  table : 'ow',
+  cartocss: CARTOCSS
+});
 ```
 
-
-`getValueForPos(x, y[, step])`
+`getValueForPos(x, y[, step])` |
```

Comment:
This table looks a bit weird.
![screen shot 2015-11-04 at 11 25 58](https://cloud.githubusercontent.com/assets/390398/10935506/de6db7ba-82e6-11e5-808d-449cee57891d.png)

I'd explain each method separately with the same format we use for other methods.